src/utils/DependencyUtils.ts

```diff
@@ -1,3 +1,26 @@
+import Game from "../model/game/Game";
+import ManifestV2 from "../model/ManifestV2";
+import ThunderstoreCombo from "../model/ThunderstoreCombo";
+import ThunderstoreVersion from "../model/ThunderstoreVersion";
+import { getCombosByDependencyStrings } from '../r2mm/manager/PackageDexieStore';
+
+export enum InstallMode {
+    INSTALL_SPECIFIC = 0,
+    UPDATE_ALL = 1
```

Comment:
This might or might not be slighlty misleading, as this only uses the latest version for the dependencies, but keeps the original version(s) of the input combo(s). Maybe adding a docstring comment explaining this would be good.
Or maybe the actual issue here is the "- `installMode` should be `INSTALL_SPECIFIC`". Saying it "should be" is a bit vague. As in "it should be, but feel free to use the other one" or "it should be, because the code doesn't currently use it in any other way than this". 🤔